Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -519,6 +519,29 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   virtual void SetSliceResolutionMode(int mode);
   vtkGetMacro(SliceResolutionMode, int);
 
+  enum SliceReconstructionType
+  {
+    SliceReconstructionMax=0,
+    SliceReconstructionMin,
+    SliceReconstructionMean,
+    SliceReconstructionSum
+  };
+
+  ///
+  /// method for setting slab visibility
```

Comment:
Considering grouping the docstring and improve the wording. For example:
* https://github.com/Slicer/Slicer/blob/5e460add5b9163fb2f80e33037624c97f5b4d7f4/Modules/Loadable/Segmentations/MRML/vtkMRMLSegmentEditorNode.h#L97-L137